crypto/Cargo.toml

```diff
@@ -49,11 +50,11 @@ harness = false
 default = []
 celo = ["ark-ec", "ark-ff", "ark-serialize", "ark-bls12-377", "ark-std", "celo-bls"]
 copy_key = []
+zero_crypto = []
```

Comment:
I'd change the name to something else, cause zero_crypto might be confused with zero knowledge proofs.